cmd/build-uki.go

```diff
@@ -93,7 +93,8 @@ func NewBuildUKICmd() *cobra.Command {
 				return fmt.Errorf("keys directory does not exist: %s", keysDir)
 			}
 			// Check if the keys directory contains the required files
-			requiredFiles := []string{"db.der", "db.key", "db.auth", "KEK.der", "KEK.auth", "PK.der", "PK.auth", "tpm2-pcr-private.pem"}
+			//requiredFiles := []string{"db.der", "db.key", "db.auth", "KEK.der", "KEK.auth", "PK.der", "PK.auth", "tpm2-pcr-private.pem"}
+			requiredFiles := []string{"db.auth", "KEK.auth", "PK.auth"}
 			for _, file := range requiredFiles {
```

Comment:
should we precheck here if the customcerts dir is empty and fail before continuing?

Reply:
We can yes. I'll add a check. I'll just check that the directory exists and it has at least the db and KEK files. If the file formats are not correct or anything like that, I'll let it fails later.

Reply:
sorry, my mistake, this is `build-uki`, it doesn't take any "custom certs" argument. You pass the "custom certs" when you call `genkey`. The `build-uki` command just accepts what `genkey` generated.